src/main/java/mate/academy/springboot/dto/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.springboot.dto;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+import mate.academy.springboot.validation.FieldMatch;
+import org.hibernate.validator.constraints.Length;
+
+@FieldMatch.List({
+        @FieldMatch(
+                field = "password",
+                fieldMatch = "repeatPassword",
+                message = "Passwords do not match!"
+        )
+})
```

Comment:
```suggestion
@FieldsValueMatch(
        field = "password",
        fieldMatch = "verifyPassword",
        message = "Password mismatch"
)
```
you can simplify  it (check next comment)